nifi-nar-bundles/nifi-standard-services/nifi-hbase-client-service-api/pom.xml

```diff
@@ -31,5 +31,16 @@
             <artifactId>nifi-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
I think these dependencies can be removed since the property descriptor now moved out of the API ?